components/backup-stream/src/router.rs

```diff
@@ -708,31 +709,29 @@ impl TempFileKey {
         }
     }
 
-    /// path_to_log_file specifies the path of record log.
+    /// path_to_log_file specifies the path of record log for v2.
     /// ```text
-    /// v1/${date}/${hour}/${store_id}/t00000071/434098800931373064-f0251bd5-1441-499a-8f53-adc0d1057a73.log
+    /// v2/${date}/${hour}/${store_id}/434098800931373064-f0251bd5-1441-499a-8f53-adc0d1057a73.log
```

Comment:
maybe keep some comments about v1, and the reason we use v2. and how to compatible to with v1 in if cluster upgrade.